packages/grafana-ui/src/components/Button/Button.mdx

```diff
@@ -1,10 +1,17 @@
 import { Meta, Preview, ArgsTable } from '@storybook/addon-docs/blocks';
 import { Button, LinkButton } from './Button';
+import { Alert } from '../Alert/Alert';
 
 <Meta title="MDX|Button" component={Button} />
 
 # Button
 
+<Alert severity="warning" title={'Please note:'}>
+  After reviewing this component we are asking you to use the IconButton when you require a button with only an icon.
+</Alert>
+
+When using a button please always follow a11y rules and make sure the context in which the button is located is also communicated by screen readers. If necessary add an aria label.
```

Comment:
**Nit:** maybe it would make sense to list / link some of the a11y rules one should pay attention to in this scenario?

Reply:
Sure, I added a link to the related WCAG recommendation.